src/components/EventCard/EventCard.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Card } from 'antd';
+import './styles.scss';
+import eventCover from '../../assets/eventcard__event-cover.png';
+import downArrow from '../../assets/eventcard__down-arrow.svg';
+import pointsCircle from '../../assets/eventcard__points-circle.svg';
+
+const { Meta } = Card;
+
+const EventCard = () => {
```

Comment:
So, one thing I should probably show you next week is how to use props,

Basically, instead of setting your src with these imports, we're going to use props in order to make this component reusable